apps/backend/src/router/game.routes.ts

```diff
@@ -189,6 +190,28 @@ router.get('/init', async (req: Request, res: Response) => {
   } else {
     // all good - send game response
     if (response.game.finished) {
+      if (response.game.guessed) {
+        // previous game was finished, compare score with highest
+        const timePlayed =
+          Number(response.game.timestamp_finish) -
+          Number(response.game.timestamp_start);
+        const score = response.game.score;
+        // check with previous
+        if (
+          !response.highest ||
+          score > response.highest.score ||
+          (score === response.highest.score &&
+            timePlayed >= response.highest.timeMs)
+        ) {
```

Comment:
Maybe it would be beneficial to have this condition put in a function?


```suggestion
        const isNewHighScore = ():boolean => (
          !response.highest ||
          score > response.highest.score ||
          (score === response.highest.score &&
            timePlayed >= response.highest.timeMs)
        );
        if (isNewHighScore()) {
```